src/convertToObject.js

```diff
@@ -6,7 +6,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
+    .filter((line) => line.trim())
+    .map((line) => line.split(/:(.+)/).map((part) => part.trim()))
+    .reduce((styles, [key, value]) => {
```

Comment:
The `reduce` function is correctly used to accumulate the key-value pairs into an object. Ensure that the initial value `{}` is correctly set to avoid any issues with undefined values.